live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/hx_insight/storage/connector/PreSignedUrlRequester.java

```diff
@@ -79,8 +82,9 @@ public void configure()
 
         from(LOCAL_ENDPOINT)
             .id(ROUTE_ID)
+            .setProperty(USER_AGENT_DATA, applicationInfoProvider::getUserAgentData)
             .process(authService::setHxIAuthorizationHeaders)
-            .to(integrationProperties.hylandExperience().storage().location().endpoint())
+            .toD(integrationProperties.hylandExperience().storage().location().endpoint() + ApplicationInfoProvider.USER_AGENT_PARAM)
```

Comment:
Can't we continue to use `to` here? This will always evaluate to the same endpoint during the application lifetime.

Reply:
Aha! The constant is set to `"&userAgent=${exchangeProperty.%s}".formatted(USER_AGENT_DATA)` - ok :+1: